examples/providers/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@gensx-examples/providers",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18.0.0"
+  },
+  "scripts": {
+    "dev": "nodemon",
+    "start": "NODE_OPTIONS='--enable-source-maps' tsx ./index.tsx",
+    "build": "tsc",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "test": "NODE_OPTIONS='--enable-source-maps' tsx ./index.tsx"
```

Comment:
Looks like we should remove this?

Reply:
oh yeah, we should remove that from all the examples. pushed the changes